session.go

```diff
@@ -462,6 +453,34 @@ func (s *session) WriteBytesArray(pkgs ...[]byte) (int, error) {
 	return wlg, nil
 }
 
+func sessionTimerLoop(_ gxtime.TimerID, _ time.Time, arg interface{}) error {
```

Comment:
change the name to heartbeat?  the function name has no meaning

Reply:
@wenxuwan agree with u. @georgehao pls change this func name.